### PR TITLE
[Criminal Activity] Ability to NPC's Hostage

### LINKED ISSUE #1481: some way to take NPCs hostage
so u don't always have to find a real person to be a hostage, and often times that leads to annoying 'fake' hostages

here are a couple scripts to try:

- [ ] https://legacystudio.tebex.io/package/4883736 (not free)
- [ ] https://github.com/rubbertoe98/FiveM-Scripts/tree/master/TakeHostage (free)
- [ ] https://github.com/papabendi/hostage (free)

I'm thinking maybe we can try the first one and see how it is and maybe modify it to work a bit smoother than typing a command to open that menu... or maybe just leave it if not...

and the second one I didn't try out, but I think it's like the one from the preview in the 3rd listed script

and we can at least grab the animations from it to make something custom perhaps if anything

### PR DESCRIPTION
**Pull request by maxgestic**
Ability to take NPC's hostage by aiming at them, if they don't flee then Hold E to rob and Hold G to take hostage, controls will show as a help marker

Closes #1481

### CODE REVIEW

File: resources/robnpc/sv_robnpc.lua [hunk 1]
```diff
@@ -104,9 +104,9 @@ local items = {
 
 
 AddEventHandler('Mugging:GiveReward', function(securityToken)
-    if not exports['salty_tokenizer']:secureServerEvent(GetCurrentResourceName(), source, securityToken) then
-		return false
-	end
+ --    if not exports['salty_tokenizer']:secureServerEvent(GetCurrentResourceName(), source, securityToken) then
```
Comment (minipunch) [inline resources/robnpc/sv_robnpc.lua:107]: rip

File: resources/robnpc/cl_robnpc.lua [hunk 2]
```diff
@@ -42,54 +63,140 @@ function canRobOtherPed(me, targetPed)
...
+                                TriggerServerEvent("911:MuggingNPC", x, y, z, lastStreetNAME)
+                                while GetGameTimer() - beginTime < ROB_TIME and not canceled do
+                                    exports.globals:DrawTimerBar(beginTime, ROB_TIME, 1.42, 1.475, 'ROBBING')
+                                    Wait(0)
+                                end
+                                robbing = false
+                                if canceled then
+                                    canceled = false
+                                else
+                                    -- while securityToken == nil do
```
Comment (minipunch) [inline resources/robnpc/cl_robnpc.lua:104]: rip

**Review by minipunch — CHANGES_REQUESTED** [review]

File: resources/robnpc/sv_robnpc.lua (hunk 1 above)
Comment (maxgestic) [reply resources/robnpc/sv_robnpc.lua]: fixed

File: resources/robnpc/cl_robnpc.lua (hunk 2 above)
Comment (maxgestic) [reply resources/robnpc/cl_robnpc.lua]: fixed

**Review by minipunch — APPROVED** [review]

**Pull request merged**